[2023-03-30 20:15:02,673] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2023-03-30T20:10:00+00:00 [queued]>
[2023-03-30 20:15:02,687] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2023-03-30T20:10:00+00:00 [queued]>
[2023-03-30 20:15:02,688] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-03-30 20:15:02,688] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-03-30 20:15:02,688] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-03-30 20:15:02,727] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_finalizados> on 2023-03-30 20:10:00+00:00
[2023-03-30 20:15:02,748] {standard_task_runner.py:52} INFO - Started process 8016 to run task
[2023-03-30 20:15:02,777] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_finalizados', 'scheduled__2023-03-30T20:10:00+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmpvg59vccs', '--error-file', '/tmp/tmpe5tozd9s']
[2023-03-30 20:15:02,783] {standard_task_runner.py:80} INFO - Job 88: Subtask popula_elastic_finalizados
[2023-03-30 20:15:02,927] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2023-03-30T20:10:00+00:00 [running]> on host 20743fec282a
[2023-03-30 20:15:03,087] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_finalizados
AIRFLOW_CTX_EXECUTION_DATE=2023-03-30T20:10:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-30T20:10:00+00:00
[2023-03-30 20:15:03,122] {base.py:271} INFO - GET http://192.168.0.100:9200/ [status:200 request:0.031s]
[2023-03-30 20:15:07,287] {base.py:271} INFO - HEAD http://192.168.0.100:9200/_template/desk_finalizados [status:200 request:0.007s]
[2023-03-30 20:15:07,888] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2023-03-30 20:15:08,233] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:0.250s]
[2023-03-30 20:15:08,279] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:0.222s]
[2023-03-30 20:15:08,493] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:0.326s]
[2023-03-30 20:15:08,666] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:0.336s]
[2023-03-30 20:15:08,674] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:0.216s]
[2023-03-30 20:15:08,918] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:0.272s]
[2023-03-30 20:15:10,703] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:1.893s]
[2023-03-30 20:15:10,927] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:1.933s]
[2023-03-30 20:15:11,187] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:2.094s]
[2023-03-30 20:15:11,205] {base.py:271} INFO - POST http://192.168.0.100:9200/_bulk [status:200 request:2.081s]
[2023-03-30 20:15:11,293] {logging_mixin.py:115} INFO - 0       30-03-2023
1       30-03-2023
2       30-03-2023
3       30-03-2023
4       30-03-2023
           ...    
4772    28-02-2023
4773    28-02-2023
4774    28-02-2023
4775    28-02-2023
4776    28-02-2023
Name: DataFinalizacao, Length: 4777, dtype: object
[2023-03-30 20:15:11,328] {python.py:173} INFO - Done. Returned value was: None
[2023-03-30 20:15:11,368] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_finalizados, execution_date=20230330T201000, start_date=20230330T201502, end_date=20230330T201511
[2023-03-30 20:15:11,586] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-30 20:15:11,779] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
